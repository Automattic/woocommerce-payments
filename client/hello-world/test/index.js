/** @format */
/**
 * External dependencies
 */
import renderer from 'react-test-renderer';

/**
 * Internal dependencies
 */
import { HelloWorld } from '../';

describe( 'HelloWorld', () => {
	test( 'it renders correctly', () => {
		const tree = renderer.create( <HelloWorld /> ).toJSON();
		expect( tree ).toMatchSnapshot();
	} );
} );
