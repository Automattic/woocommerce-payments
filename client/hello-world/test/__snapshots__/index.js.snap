// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld it renders correctly 1`] = `
<div
  className="woocommerce-payments__section"
>
  Hello, world.
</div>
`;
